docs/scenarios_authoring/HandlingSchemas.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 11
+---`
+
+# Handling Schemas
+
+## Overview
+
+Nussknacker supports two types of schema:
+* [JSON Schema](https://json-schema.org/) in version: `Draft 7`
+* [Avro Schema](https://avro.apache.org/) in version: `1.11.0`
+
+Avro Schema and JSON Schema are available on [Streaming Source/Sink](/docs/scenarios_authoring/DataSourcesAndSinks.md)
+in contrast to [RequestResponse Source/Sink](/docs/scenarios_authoring/RRDataSourcesAndSinks.md) where available is just only one - JSON Schema.
+
+Nussknacker converts `source schema` to its own static `Typing` information to create hinting on the FE Designer,
```

Comment:
I think the internal of what is happening _("converts it to its own static Typing information)_ is not necessary. I would just write that is used to generate field (property?) hints and validate expressions; may be (but not necessarily) explain that output schema is used only to validate expressions used to create output record. 

FE Designer --> Designer